### PR TITLE
[16.0][MIG] fetchmail_attach_from_folder

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by thomaspaulb** [comment]
@NL66278 Why the auto_backup fix? 16.0 is currently green also without that; so maybe it should be in a separate PR?

**Comment by zamberjo** [comment]
Hi, better omit **auto_backup** changes, if you want do another PR

File: fetchmail_attach_from_folder/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- Holger Brunn <hbrunn@therp.nl>
+- Ronald Portier <ronald@therp.nl>
```
Comment (zamberjo) [inline fetchmail_attach_from_folder/readme/CONTRIBUTORS.md:2]: This module was started by @gurneyalex in 7.0, I think you should continue to respect the authorship.

---

technical and functional review, just the subject I mentioned above

**Review by anmarmo1 — APPROVED** [review]
Review functional is ok

File: fetchmail_attach_from_folder/readme/CONTRIBUTORS.md (hunk 1 above)
Comment (NL66278) [reply fetchmail_attach_from_folder/readme/CONTRIBUTORS.md:2]: The module was actually started by my former collegue @hbrunn , but as @gurneyalex  contributed some improvements I have no problem with adding him as a contributor.

**Comment by NL66278** [comment]
@zamberjo I put auto_backup changes in own PR now: https://github.com/OCA/server-tools/pull/2939

**Review by zamberjo — APPROVED** [review]

**Pull request merged**